code/script.js

```diff
@@ -1,13 +1,85 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById("chat")
+const nameForm = document.getElementById("name-form")
+const inputwrapper = document.getElementById("input-wrapper")
 
 // Global variables, if you need any, declared here
 
 // Functions declared here
 
+const nameInput = document.getElementById("name-input")
+
+const handleNameInput = (event) => {
+  event.preventDefault()
+
+  showMessage(nameInput.value, "user")
+
+  nameInput.value = ""
+
+  setTimeout(()=> question1(), 1000)
+
+  
+
+  }
+
+  const question1 = () => {
+    showMessage("Hur ofta vill du spara?", "bot")
```

Comment:
If you sent a parameter along when you invoked the function, like this:
`setTimeout(()=> question1(nameInput.value), 1000)`
you could pick it up here and use it in your message:
```
const question1 = name => {
  showMessage(`Härligt ${name}! Hur ofta vill du spara?`, "bot")
```